zad1/main.rb

```diff
@@ -1,13 +1,14 @@
 def sort_letters(string)
-
+    string.chars.sort.join
```

Comment:
To oczywiście nie zmienia samego działania, ale przyjęło się, że [w rubym używamy 2 spacji do wcięć](https://stackoverflow.com/questions/6741586/rails-coding-standards-why-2-space-indentation).

Reply:
Dzięki, będę o tym pamiętać w przyszłości.